src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+
+    public static void doWork() {
+        System.out.println("Excavator started its work ");;
```

Comment:
There is an unnecessary semicolon at the end of the `System.out.println` statement. Please remove the extra semicolon to clean up the code.